ext/graphql_libgraphqlparser_ext/extconf.rb

```diff
@@ -1,6 +1,11 @@
 require 'mkmf'
 
-dir_config('graphql', '/usr/local/include/graphqlparser', '/usr/local/lib')
-abort 'missing libgraphqlparser' unless have_library 'graphqlparser'
+prefixes = %w(/usr /usr/local)
+prefix = prefixes.find{ |prefix| Dir["#{prefix}/lib/libgraphqlparser*"].first }
+abort 'missing libgraphqlparser' unless prefix
```

Comment:
updated to call `dir_config('graphql')` if a prefix isn't found.  I tested it with libgraphqlparser installed in /opt/libgraphqlparser and adding `ext.config_options << '--with-graphql-include=/opt/libgraphqlparser/include/graphqlparser --with-graphql-lib=/opt/libgraphqlparser/lib'` to the `Rake::ExtensionTask.new "graphql_libgraphqlparser_ext" do |ext|` block in the Rakefile.